File: sample-java-jpa/src/main/java/hq/sombriks/sample/model/PartyStatus.java
package hq.sombriks.sample.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Data
@Entity
@Table(name = "party_status")
public class PartyStatus {

  public PartyStatus() {
  }

  public PartyStatus(int id) {
    this.id = id;
  }

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "party_status_id")
  private Integer id;

  @Column(name = "party_status_description")
  private String description;

}

File: sample-javascript-sequelize/src/model/PartyStatus.js
// model/PartyStatus.js
const Sequelize = require("sequelize");
const {sequelize} = require("../db");

exports.PartyStatus = sequelize.define(
  "PartyStatus",
  {
    id: {
      type: Sequelize.INTEGER,
      primaryKey: true,
      field: "party_status_id",
    },
    description: {
      type: Sequelize.STRING,
      field: "party_status_description",
    },
  },
  {tableName: "party_status", timestamps: false},
);
